azext_iot/_params.py

```diff
@@ -495,6 +496,44 @@ def load_arguments(self, _):
             "Account and Contributor role for the IoT Hub.",
         )
 
+    with self.argument_context("iot hub device-identity generate-mqtt-credentials") as context:
+        context.argument(
+            "connection_string",
+            options_list=["--connection-string", "--cs"],
+            help="Target connection string. This bypasses the IoT Hub registry and generates the user credentials directly "
+                 "from the supplied symmetric key without further validation. All other command parameters aside from "
+                 "password-creation-time and password-expiry-in-secs will be ignored. Supported connection string types: Device, "
+                 "Module."
+        )
+        context.argument(
+            "password_expiry_in_secs",
+            options_list=["--password-expiry-in-secs", "--pes"],
+            type=int,
+            help="Valid password duration in seconds.",
+        )
+        context.argument(
+            "password_creation_time",
+            options_list=["--password-creation-time", "--pct"],
+            type=int,
+            help="Valid time in seconds for password creation.  Default:  Current time.",
+        )
+        context.argument(
+            "product_info",
+            options_list=["--product-info", "--pi"],
+            help="Description of the product.",
+        )
+        context.argument(
+            "dtmi",
+            options_list=["--dtmi"],
+            help="Digital Twins model Id. Example: dtmi:com:example:Room;2",
+        )
+        context.argument(
+            "version",
+            options_list=["--format-version", "--fv"],
+            help="MQTT Connect Credentials Format Version to use when generating the credentials. Supported values: v1, v2."
+                 " Default: v2",
```

Comment:
We shouldn't need to indicate the default text here if a default value is assigned to the respective command handler input.

Also would be good idea to use an enum instead of a string.

Reply:
"MQTT connection credentials format version to use ... ..."